src/docs/development/platform-integration/ios-app-clip.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Adding an iOS App Clip target
+description: How to add an iOS 14 App Clip target to your Flutter project
+---
+
+{{site.alert.important}}
+This support is offered experimentally, may not be stable, and may change
+in the future.
+{{site.alert.end}}
+
+This guide describes how to manually add another Flutter-rendering iOS App Clip
+target to your existing Flutter project or [add-to-app][] project.
+
+For discussions on a feature request to automatically integrate, see
+[#65451][].
+
+{{site.alert.warning}}
+This is an advanced guide and is best intended for audience with a working
+knowledge of iOS development.
+{{site.alert.end}}
+
+{{site.alert.warning}}
+CocoaPods version 1.10.0.beta.1 or higher is required to run Flutter apps with
+plugins.
+{{site.alert.end}}
+
+The result of this guide is also demonstrated in a [sample project][].
+
+
+## Step 1 - Open project
+
+Open your iOS Xcode project, such as `ios/Runner.xcworkspace` for full-Flutter
+apps.
```

Comment:
Let me know if this is sensible. Generally, this guide offers 2 levels of details for 2 target audience groups. The full-Flutter crowd might know nothing about iOS development so it's a bit more hand holding. The add-to-app crowd knows about iOS development already by definition so the instructions are more handwavy. Though technically, we can't offer hand-held instructions for add-to-app either since we can't know the details of their project (their project location in this case). 

For the more savvy add-to-app crowd, they'll know exactly what "Open your iOS Xcode project" means.

Reply:
ok